src/renderer/shared/hocs/withErrorToast.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+
+import { showErrorToast } from 'shared/actions/toastsActions';
+
+export default function withErrorToast(propName = 'showToast') {
```

Comment:
or either pass in for each `withXXXToast` it's specific propName aka `errorToast` for the `withErrorToast` hoc